app/controllers/foreman_salt/minions_controller.rb

```diff
@@ -6,14 +6,21 @@ class MinionsController < ::ForemanSalt::ApplicationController
     add_smart_proxy_filters :node, :features => 'Salt'
 
     def node
-      enc = {}
+      enc = {'foreman' => {}}
```

Comment:
Why? This breaks the foreman-node script, and isn't the data format expected by the salt master.